processor/README.md

```diff
@@ -71,3 +83,48 @@ processors:
 ```
 Refer to [config.yaml](attributes/testdata/config.yaml) for detailed
 examples on using the processor.
+
+## <a name="node-batcher"></a>Node Batcher Processor
+<FILL ME IN - I'M LONELY!>
+
+## <a name="probabilistic-sampler"></a>Probabilistic Sampler Processor
+<FILL ME IN - I'M LONELY!>
+
+## <a name="queued"></a>Queued Processor
+<FILL ME IN - I'M LONELY!>
+
+## <a name="span"></a>Span Processor
+The span processor modifies top level settings of a span. Currently, only 
+renaming a span is supported. 
+
+### Name a span
+It takes a list of `keys` and an optional `separator` string. The attribute
+value for the keys are used to create a new name in the order specified in the
+configuration. If a separator is specified, it will separate values.
+
+If renaming is dependent on attributes being modified by the `attributes`
+processor, ensure the `span` processor is specified after the `attributes`
+processor in the `pipeline` specification.
+
+For more information, refer to [config.go](span/config.go)
+```yaml
+span:
+  name:
+    # Keys represents the attribute keys to pull the values from to generate the
+    # new span name.
+    keys: [<key1>, <key2>, ...]
```

Comment:
Would it be more self-documenting if we use `attributes` instead of `keys` here?
Maybe even `from-attributes`? (But that may be too verbose, not sure).